Update to newest icons in Chrome and Firefox.

lucyhe opened the pull request:
Ran grunt test.

This updates icons in Chrome and FF to the newest icons. Chrome does not show the animation in the toolbar, but it does in the popup (the giving and getting circle-arrow and triple-arrow).
Addressing: https://github.com/uProxy/uproxy/issues/471 and https://github.com/uProxy/uproxy/issues/849

Also deleted unused icons and updated the notification icon:
![image](https://cloud.githubusercontent.com/assets/8723127/6141553/b7cf9382-b175-11e4-9b36-0caa0a43157d.png)

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/uproxy/uproxy/897)
<!-- Reviewable:end -->

dborkan commented on src/chrome/extension/scripts/chrome_core_connector.spec.ts:
```diff
@@ -122,7 +122,7 @@ describe('core-connector', () => {
       });
       expect(chrome.browserAction['setIcon']).toHaveBeenCalledWith(
-        {path: "icons/offline-19.png"});
+        {path: "icons/NotLoggedIn_32.gif"});
```
Is it possible to refer to the constants from UserInterface here, and in other files where you are writing out the full filename+path?

lucyhe commented:
I tried to use the same string constants in Polymer code but it was only working with Chrome. The Polymer binding didn't seem to be working correctly in Firefox. For example, for buddySharingIcon='icon.gif' <img src="{{buddySharingIcon}}"></img> shows a broken image for Firefox, while <img src="icon.gif"></img> works correctly.

Also added to this pull request:
- deleted ui_enums.ts and moved it's contents to the UI module in ui.ts

PTAL, thanks!

lucyhe commented:
Ah! @salomegeo said it has to do with how Firefox binds strings, will try something ...

lucyhe commented:
Still can't get around the bug for Firefox. Ideally I'd like to get the constants to work, but I think this is okay for now. PTAL, thank you!

dborkan commented:
:+1: - once we start working on uproxy-for-cloud, we will refactor the code in the background page (and firefox equivalent), and can find a good place for these constants then.
